src/editorManager.js

```diff
@@ -1,9 +1,9 @@
 import { CellCoords } from './3rdparty/walkontable/src';
 import { KEY_CODES, isMetaKey, isCtrlMetaKey } from './helpers/unicode';
 import { stopImmediatePropagation, isImmediatePropagationStopped } from './helpers/dom/event';
-import { getEditorInstance } from './editors';
+import { getEditorInstance } from './editors/editors';
```

Comment:
The same as [here](https://github.com/handsontable/handsontable/pull/7403/files#diff-8ab41fe13597e1554b5d6b4c227b5f123ff2d6726a7f3688a8b8d1224fe1d4f3R20).

Reply:
I'll reply there